### ISSUE TITLE
Nice!

### ISSUE THREAD

**Issue by maphouse** [geog-464/lab8-sophieee666#1]
Excellent, it seems like you've set out to build something that could be of use to yourself and others in your field, on a topic I admit to not know much about! Just make sure you contextualize the tool in your documentation as much as possible, how it can be used and what are its limits. 

Assuming you will be manipulating images, you may want to use rasterio (py package for handling raster data), but there are also several other (non-geographic) image processing libraries that you might find more useful instead which we could discuss.

Do you have the data you're talking about? Is it cleaned up?

I look forward to seeing what you come up with...

**Comment by sophieee666** [comment]
@maphouse 

Yeah, my background is in geology and I have been studying GIS/coding specifically to learn how to apply it to my field.

My tool's use will be somewhat limited for multiple reasons: I am still a coding noob, it won't be able to anticipate all potential input data idiosyncrasies, the breadth of possible analyses would take months to code, etc. I also understand that geology tools are pretty niche. So, I will definitely take care to properly contextualize my tool and hopefully outline some functionality gaps to be filled in any future versions.

Re: input data format - it actually will not be images/rasters (although that is a tool I came across recently that I thought was so cool). Instead, I will use stratigraphic sequence logs, which I have been able to find in .csv format. Basically, each row is a distinct rock layer within the greater rock formation, and the columns contain depth and grain size information. 

The .csv file I will use for my demo has missing data, so part of my tool's functionality will involve cleaning/anticipating "no data" values.

I hope that my toolkit will be comprehensive enough to make up for the fact that the topic is not strictly geo-spatial. Let me know what you think.

**Comment by maphouse** [comment]
Very sad to see you dropped the course! You were doing very well. I hope it wasn't due to anything too serious in your personal life. All the best!
